edx_proctoring/api.py

```diff
@@ -899,10 +899,17 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
             return_course_info=True
         )
 
-        send_proctoring_attempt_status_email(
-            exam_attempt_obj,
-            credit_state.get('course_name', _('your course'))
-        )
+        if credit_state:
+            send_proctoring_attempt_status_email(
+                exam_attempt_obj,
+                credit_state.get('course_name', _('your course'))
+            )
+        else:
+            log.error(
+                "Could not retrieve the credit state for the user with the user id %r in the course %r.",
```

Comment:
Are we fixing an actual issue?  It's unclear if this should be an error, warning, or info.  What are the cases that the credit state service would not be available?

Also, the message seems misleading, because we never even get this far.  Maybe:
```
The credit state service is not available.  Skipping proctoring attempt status email for the user with the user id %r in the course %r.
```
?